<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_trapezoid</class>
 <widget class="QDialog" name="Dialog_trapezoid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Dialog_trapezoid
{
background-color: #101223;
}

QTableWidget::horizontalHeaderItem{
background-color: black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tab_rect">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>52</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="autoScrollMargin">
      <number>10</number>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>153</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <row>
      <property name="text">
       <string>Значение</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Левая сторона</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Правая сторона</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Меньшее основание</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Большее основание</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Высота</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Площадь</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Периметр</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <widget class="QTableWidget" name="table_ang">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>52</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>266</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Значение</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Левый нижний угол</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Левый верхний угол</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Правый верхний угол</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Правый нижний угол</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
